examples/plotting/plot_rotated_rectangle.py

```diff
@@ -0,0 +1,62 @@
+"""
+============================
+Drawing a rotated rectangle on a map
+============================
+
+This example will demonstrate how to draw a rectangle that can be rotated relative
+to the axes on a map using :meth:`~sunpy.map.GenericMap.draw_quadrangle`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, SkyOffsetFrame
+
+import sunpy.data.sample
+import sunpy.map
+
+################################################################################
+# Let's start with a sample AIA image.
+
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+# sphinx_gallery_defer_figures
+
+fig = plt.figure(figsize=(5, 5))
```

Comment:
Why this figure size?

Reply:
I just checked out the plot_rectangle.py file and used the same figure size.

Reply:
Can you remove it and check the output difference.

Reply:
the output is the same, just the image is resized.